server/middleware/auth.js

```diff
@@ -0,0 +1,37 @@
+const passport = require('passport')
+const passportJWT = require('passport-jwt')
+const ExtractJwt = passportJWT.ExtractJwt
+const Strategy = passportJWT.Strategy
+
+// load up the user model
+const User = require('../src/user/model')
+const config = require('../config/auth') // get auth secret key
```

Comment:
Inline comments are not readable and are ugly... better put a new line and add the comment before.